packages/app/components/OrdersProgressBar.tsx

```diff
@@ -0,0 +1,20 @@
+import { OrderProps, totalOrdersDone, totalOrders } from "@/app/models/order";
+import React, { useRef, useEffect } from "react";
+
+export const OrdersProgressBar = ({ order }: OrderProps) => {
```

Comment:
Should we do the same with this component? Seems to be a very custom component that can be defined inside `StackProgress`.

Reply:
Actually this one will be used on other parts, that's why I add it to components. For eg. when we do the custom OrderCards on mobile.

<img width="255" alt="image" src="https://github.com/SwaprHQ/stackly-ui/assets/5664434/36da5c14-103b-4474-a7d3-1029fcc148b3">